testclean/suite_test.go

```diff
@@ -63,9 +62,7 @@ func init() {
 func TestAPIs(t *testing.T) {
 	RegisterFailHandler(Fail)
 
-	RunSpecsWithDefaultAndCustomReporters(t,
-		"Controller Suite",
-		[]Reporter{printer.NewlineReporter{}})
+	RunSpecs(t, "Controller Suite")
```

Comment:
is this related to this PR (upgrade ? )

Reply:
Yes
RunSpecsWithDefaultAndCustomReporters is deprecated replace by RunSpecs.